app/mailers/app_mailer.rb

```diff
@@ -7,4 +7,9 @@ def reset_password(user)
     @user = user
     mail from: 'info@myflix.com', to: user.email, subject: 'Your password reset link'
   end
+
+  def send_invite(invite)
+    @invite = invite
+    mail from: 'info@myflix.com', to: invite.friend_email, subject: '#{@invite.user.name} has invited you to join MyFlix'
```

Comment:
you can use `default from: "..."` to refactor the `from: '...'`. it works similarly as `before_action`